recipes/pcst-fast/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pcst-fast" %}
+{% set version = "1.0.10" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/d4/fd/64b51c867bad63e6622ff97cad6230b94b19b5a61e30424cd69c8353091c/pcst_fast-{{ version }}.tar.gz
+  sha256: 3b5694110ce2e004471f383267d5e4ab7fe1ba9828954e8c42560ac1e42b25e6
+
+build:
+  number: 0
+  skip: True  # [py < 38]
+  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps --use-pep517 -vvv
+  run_exports:
+    - {{ pin_subpackage('pcst-fast', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - pip
+    - pybind11 >=2.4
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - pybind11 >=2.4
+    - python
+
+test:
+  commands:
+    - python --version
+
+about:
+  home: https://github.com/fraenkel-lab/pcst_fast
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

As is, this package is vendoring `pybind11`, `googletest`, `googlemock` under the `external/` directory. Either those need to be removed and the package strictly built with Conda Forge versions, or the licenses need to be explicitly bundled. In the latter case, the recipe should change to:

```suggestion
  license: MIT AND BSD-3-Clause AND Apache-2.0
  license_family: MIT
  license_file:
    - LICENSE.txt
    - external/pybind11/LICENSE
    - external/googletest/googletest/LICENSE
    - external/googletest/googlemock/LICENSE
    - external/googletest/googlemock/scripts/generator/LICENSE
```

---

I'd _like_ it to be packaged strictly with conda forge versions, but I assume the original author has made this decision so that it can be successfully run from a pip install. I'll go for explicit license bundling and raise an issue upstream.